source/common/tcp_proxy/BUILD

```diff
@@ -54,3 +64,57 @@ envoy_cc_library(
         "@envoy_api//envoy/extensions/filters/network/tcp_proxy/v3:pkg_cc_proto",
     ],
 )
+
+envoy_cc_library(
```

Comment:
Since this is a new file, I think we can put it directly in the includes/ directory

Reply:
moved to include/common/tcp/